### PR TITLE
[UI] Unable to deselect test directories

### PR DESCRIPTION
**Pull request by KyleJu**
Fix for #1951. Users are unable to deselect test directories in the Triage Metadata UI.

The issue is caused by the fact that the `test` is being overwritten with extra `/*` inside of `handleSelectMetadata`

### CODE REVIEW

**Comment by KyleJu** [comment]
@Hexcles The `test` shouldn't be passed by reference or a data binding. I cannot seem to understand why

**Comment by Hexcles** [comment]
Hmm what's this bug? Is this #1951 that I just filed?

**Comment by KyleJu** [comment]
> Hmm what's this bug? Is this #1951 that I just filed?

Correct

**Comment by Hexcles** [comment]
So I suspect the `path` property is passed in, not a simple string. Could you log `typeof test` and `test.prototype`?

**Comment by KyleJu** [comment]
> test.prototype



> So I suspect the `path` property is passed in, not a simple string. Could you log `typeof test` and `test.prototype`?

I debugged in the devtool and I got "string" and undefined, which seem right

**Comment by KyleJu** [comment]
Since we will do a refactoring [here](https://github.com/web-platform-tests/wpt.fyi/pull/1935#discussion_r421216680), I will add the tests in #1935

File: webapp/views/wpt-results.js [hunk 1]
```diff
@@ -1049,16 +1049,18 @@ class WPTResults extends Pluralizer(WPTColors(WPTFlags(PathInfo(LoadingState(Tes
     return (e) => {
```
Comment (Hexcles) [inline webapp/views/wpt-results.js:1049]:
```js
handleSelectMetadata() {
  // Prevent accidentally modifying these captured parameters in the closure.
  const [index, test] = arguments;
```

**Comment by KyleJu** [comment]
friendly ping @Hexcles

File: webapp/views/wpt-results.js [hunk 2]
```diff
@@ -1045,20 +1045,24 @@ class WPTResults extends Pluralizer(WPTColors(WPTFlags(PathInfo(LoadingState(Tes
     }
   }
 
-  handleSelectMetadata(index, test) {
+  handleSelectMetadata() {
     return (e) => {
+      // Prevent accidentally modifying these captured parameters in the closure.
+      const [index, test] = arguments;
```
Comment (Hexcles) [inline webapp/views/wpt-results.js:1051]: We should capture the arguments in the parent function instead.

```suggestion
    // Prevent accidentally modifying these captured parameters in the closure.
    const [index, test] = arguments;
    return (e) => {
```

**Pull request merged**